### ISSUE TITLE
Failed copying input tensor from CPU to GPU

### ISSUE THREAD

**Issue by zfeilongzaitian** [tensorflow/tensorflow#52032]
### 1. System information
When i try use bert pretrain model to do my downstream tasks(what i use to load bert pretrain model is keras-bert), but when i try load my model to CPU and use GPU to train, the mistake occured. I have two GPUs with memory 32g* 2 and CPU with 128g memory,who know this problem,please give me answer,thanks 
1. when i only use CPU, it will consumes about 112g to train;
2. when i only use GPU, the OOM kill will occur;
3. by the way, when i use bert-base pretrain model and bi-GRU net to do my classify task, what memory configuration can i have to do this task

### 2. Code
with tf.device("/cpu:0"):
   myModel = create_model()
parallel_model = multi_gpu_model(model, gpus=2)
parallel_model.compile(optimizer=Adam(1e-5), loss='categorical_crossentropy', metrics=['accuracy'])
parallel_model.fit(train_data, train_label, validation_data=(valid_data, valid_label), epochs=epoch, batch_size=8, shuffle=True, callbacks=[history])

### 3. (optional) Any other info / logs
the error log is 
![image](https://user-images.githubusercontent.com/30098191/133622256-8970595f-5cd2-4e45-ae4e-0ee29241bf77.png)

**Comment by mohantym** [comment]
Hi @zfeilongzaitian !Could you please share a complete stand alone  code with links to Datasets as a  Colab gist to replicate the above issue. It will help expedite the issue.

**Comment by zfeilongzaitian** [comment]
> Hi @zfeilongzaitian !Could you please share a complete stand alone code with links to Datasets as a Colab gist to replicate the above issue. It will help expedite the issue.

sorry replay for so later, I found the reason,because i make a batch_size too big, so it use too much memory

**Comment by mohantym** [comment]
ok @zfeilongzaitian , Thanks for replying! Please Feel free to close this one and create a new one if you face any problem. Thanks!
